File: Basics_with_JavaScript/78 Conditional Statements Advanced/More Exersises/07. School Camp.js
function vladi(input){

    let season = input[0];
    let groupType = input[1];
    let studentQty = Number(input[2]);
    let nightsQty = Number(input[3]);
    let sport = "";
    let nightPice = 0;

    switch (season) {
        case "Winter":
            switch (groupType) {
                case "girls":
                    nightPice = 9.60;
                    sport = "Gymnastics"
                    break;
                case "boys":
                    nightPice = 9.60;
                    sport = "Judo"
                    break;
                case "mixed":
                    nightPice = 10;
                    sport = "Ski"
                    break;
                default:
                    break;
            }
            break;

        case "Spring":
            switch (groupType) {
                case "girls":
                    nightPice = 7.20;
                    sport = "Athletics"
                    break;
                case "boys":
                    nightPice = 7.20;
                    sport = "Tennis"
                    break;
                case "mixed":
                    nightPice = 9.50;
                    sport = "Cycling"
                    break;
                default:
                    break;
            }
            break;

        case "Summer":
            switch (groupType) {
                case "girls":
                    nightPice = 15;
                    sport = "Volleyball"
                    break;
                case "boys":
                    nightPice = 15;
                    sport = "Football"
                    break;
                case "mixed":
                    nightPice = 20;
                    sport = "Swimming"
                    break;
                default:
                    break;
            }
            break;
        default:
            break;
    }
    let totalNightsPrice = nightPice * nightsQty * studentQty;
    if (studentQty >= 50) {
        totalNightsPrice *= 0.50;
    } else if (studentQty >= 20 && studentQty < 50) {
        totalNightsPrice *= 0.85;
    } else if (studentQty >= 10 && studentQty < 20) {
        totalNightsPrice *= 0.95;
    }
    totalNightsPrice = (totalNightsPrice).toFixed(2);
    console.log(`${sport} ${totalNightsPrice} lv.`);
    }

    vladi(["Spring", "girls", "20", "7"]);
    vladi(["Winter", "mixed", "9", "15"]);
    vladi(["Summer", "boys", "60", "7"]);
    vladi(["Spring", "mixed", "17", "14"]);

File: Basics_with_Python/PythonBasicCourse/3_Conditional_Statements_Advanced/8_Conditional_Statements_Advanced___Exercise___Lab/more_exercises/07._School_Camp.py
season = input()
sex = input()
students = int(input())
nights = int(input())
price = 0
sport = 0
if sex == "boys":
    if season == "Winter":
        price = 9.60
        sport = "Judo"
    if season == "Spring":
        price = 7.20
        sport = "Tennis"
    if season == "Summer":
        price = 15
        sport = "Football"
elif sex == "girls":
    if season == "Winter":
        price = 9.60
        sport = "Gymnastics"
    if season == "Spring":
        price = 7.20
        sport = "Athletics"
    if season == "Summer":
        price = 15
        sport = "Volleyball"
elif sex == "mixed":
    if season == "Winter":
        price = 10
        sport = "Ski"
    if season == "Spring":
        price = 9.50
        sport = "Cycling"
    if season == "Summer":
        price = 20
        sport = "Swimming"


total = price * nights * students
if students >= 50:
    total = total * 0.50
elif 20 <= students < 50:
    total = total * 0.85
elif 10 <= students < 20:
    total = total * 0.95
print(f"{sport} {total:.2f} lv.")
